docs/apidocs/objective.rst

```diff
@@ -1,10 +1,12 @@
-.. _objective:
-
-##################
-Objective Function
-##################
+#############################################################
+Objective Function (:mod:`qiskit_addon_aqc_tensor.objective`)
```

Comment:
```suggestion
Objective function(s) (:mod:`qiskit_addon_aqc_tensor.objective`)
```
there's only one for now, but there will soon (hopefully) be two.  I'm not sure if "(s)" is a good middle ground or not.